packages/babel-preset-typescript/test/normalize-options.spec.js

```diff
@@ -0,0 +1,105 @@
+import normalizeOptions from "../src/normalize-options";
+describe("normalize options", () => {
+  describe("Babel_8_breaking", () => {
+    let old_babel_8_breaking_env;
+    beforeAll(() => {
+      old_babel_8_breaking_env = process.env.BABEL_8_BREAKING;
+      process.env.BABEL_8_BREAKING = "1";
+    });
+    afterAll(() => {
+      if (old_babel_8_breaking_env) {
+        process.env.BABEL_8_BREAKING = old_babel_8_breaking_env;
+      } else {
+        delete process.env.BABEL_8_BREAKING;
+      }
+    });
```

Comment:
Since we run tests with the `BABEL_8_BREAKING` flag on CI (and we can manually do it locally), we don't need to set it here.

I'm worried that this will fail on `make prepublish`, since in the prepublish build we strip the `BABEL_8_BREAKING` checks in the compiled code.

Reply:
Then it is pretty scary that the `make prepublish` actually passed. 😂

I will skip the test if `BABEL_8_BREAKING` does not present.

Reply:
> Then it is pretty scary that the `make prepublish` actually passed. 😂


I'm investigating :grin:

Reply:
Ok, the test is not failing because it imports `src`, so the source is compiled just-in-time by Jest regardless of what we have in `lib`.